core/dom/document.js

```diff
@@ -247,6 +247,46 @@ CKEDITOR.tools.extend( CKEDITOR.dom.document.prototype, {
 		return new CKEDITOR.dom.window( this.$.parentWindow || this.$.defaultView );
 	},
 
+	/**
+	 * Gets the DOM scrolling element for this document. Scrolling element allows for using `scrollTop` and `scrollLeft` on entire document.
+	 *
+	 * @since 4.10.0
+	 * @returns {CKEDITOR.dom.element} Scrolling element.
+	 */
+	getScrollingElement: function() {
+		var scrollingElement = this.$.scrollingElement || this.$.documentElement || this.$.body;
+		return new CKEDITOR.dom.element( scrollingElement );
+	},
+	/**
+	 * Returns scroll position of the document.
+	 *
+	 * @since 4.10.0
+	 * @returns {Object} Object representing vertical and horizontal scroll position of the document.
```

Comment:
If such object is returned in many places, I wonder about adding some virtual type for it (`CKEDITOR.dom.scrollPosition?`) – similar to `CKEDITOR.dom.rect`